### PR TITLE
Create Issue temaplate #2

### LINKED ISSUE #2: Create Issue temaplate
Greetings of the day folks !!
If anyone want's to create the issue template then feel free to comment 

The issue template should contains ->  **Bug report , Feature request  and  other .** 
like shown below 

Screenshot from 2022-07-25 17-32-13

### PR DESCRIPTION
**Pull request by surjyaneeh**
## Description

Fixes #2 

## Type of change

Please delete options that are not relevant. In order to tick the check box just put x inside them for example `[x]` like this :

- [x] Improve the readme of any project

## Checklist ✓

Please delete options that are not relevant. In order to tick the check box just put x inside them for example `[x]` like this

- [x] l have performed a self-review of my own code or work.

- [x] I have commented on my code, particularly in hard-to-understand areas.

- [x] My changes generate no new warnings

- [x] I'm HSOC'22 contributor

## Screenshots 
1. 
image

2.
image

3.
image

### CODE REVIEW

**Comment by Vikash-8090-Yadav** [comment]
@surjyaneeh  add these chnages here also that i mentioned in the web3.0-world

**Comment by surjyaneeh** [comment]
> @surjyaneeh add these chnages here also that i mentioned in the web3.0-world

Sure .

**Comment by surjyaneeh** [comment]
> @surjyaneeh add these chnages here also that i mentioned in the web3.0-world

Done.

File: .github/ISSUE_TEMPLATE/improvements.md [hunk 1]
```diff
@@ -12,10 +12,11 @@ Describe more about the change.
 
 ### Domain of Docs
 
-- [ ] Solidity
-- [ ] Python
-- [ ] JavaScript
-- [ ] Readme
+- [] Readme
```
Comment (Vikash-8090-Yadav) [inline .github/ISSUE_TEMPLATE/improvements.md:15]: Remove 16 , 17 ,and 18 and add this ->
 [ ] Solidity
[ ] Python
[ ] JavaScript
[ ] Golang

**Review by Vikash-8090-Yadav — CHANGES_REQUESTED** [review]

**Review by Vikash-8090-Yadav — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/improvements.md (hunk 1 above)
Comment (Vikash-8090-Yadav) [reply .github/ISSUE_TEMPLATE/improvements.md:15]: @surjyaneeh  update all this

**Comment by surjyaneeh** [comment]
@Vikash-8090-Yadav is it okay now?

File: .github/ISSUE_TEMPLATE/add-project.md [hunk 2]
```diff
@@ -1,22 +1,27 @@
 ---
 name: Add a Project
-about: Add Smart Contract
+about: Add a new Web3 Project
```
Comment (Vikash-8090-Yadav) [inline .github/ISSUE_TEMPLATE/add-project.md:3]: replace line 3 and add 
about: Add Smart Contract

File: .github/ISSUE_TEMPLATE/add-project.md [hunk 3]
```diff
@@ -1,22 +1,27 @@
...
+(Describe the project over here)
 
 ### Doc Domain
 <!--
 Leave it unchecked Owner will mark it :)
 -->
-- [ ] Solidity
-- [ ] Python
-- [ ] JavaScript
+- [] Solidity , Html, Css , JavaScript , Web3.js /Ether.js
```
Comment (Vikash-8090-Yadav) [inline .github/ISSUE_TEMPLATE/add-project.md:21]: remove line 21 , 22 and 23 and add 
the lines which i  mentioned in the improvements.md file

File: .github/ISSUE_TEMPLATE/add-project.md [hunk 4]
```diff
@@ -18,10 +18,10 @@ assignees: ''
...
 Leave it unchecked Owner will mark it :)
 -->
-- [] Solidity , Html, Css , JavaScript , Web3.js /Ether.js 
-- [] Solidity , ReactJs /NextJs , Web3.js /Ether.js
-- [] Solidity , Python3 , Web3.py
-- [] Other , feel free to mention below
+- [] Solidity
+- [] Python
+- [] JavaScript
+- [] Golang
```
Comment (Vikash-8090-Yadav) [inline .github/ISSUE_TEMPLATE/add-project.md:24]: after 25 add this 
[] Other , feel free to mention below

**Comment by surjyaneeh** [comment]
@Vikash-8090-Yadav done.

**Review by Vikash-8090-Yadav — COMMENTED** [review]
replace line 19  of add-project.md file with this 
And in order to tick the check box just but x inside them for example - [x] like this. Please delete options that are not relevant.

File: .github/ISSUE_TEMPLATE/improvements.md [hunk 5]
```diff
@@ -13,10 +13,10 @@ Describe more about the change.
...
 
 - [] Readme
-- [] Solidity , Html, Css , JavaScript , Web3.js / Ether.js
-- [] Solidity , ReactJs /NextJs , Web3.js /Ether.js
-- [] Solidity , Python3 , Web3.py
-- [] Other , feel free to mention below
+- [] Solidity
+- [] Python
+- [] JavaScript
+- [] Golang
```
Comment (Vikash-8090-Yadav) [inline .github/ISSUE_TEMPLATE/improvements.md:19]: after line 19 add this 
[] Other , feel free to mention below

File: .github/ISSUE_TEMPLATE/improvements.md [hunk 6]
```diff
@@ -13,10 +13,10 @@ Describe more about the change.
 ### Domain of Docs
```
Comment (Vikash-8090-Yadav) [inline .github/ISSUE_TEMPLATE/improvements.md:14]: after line 13 add this 
And in order to tick the check box just but x inside them for example - [x] like this. Please delete options that are not relevant.
as a comment

File: .github/ISSUE_TEMPLATE/improvements.md [hunk 7]
```diff
@@ -11,12 +11,15 @@ assignees: ''
 Describe more about the change.
 
 ### Domain of Docs
```
Comment (Vikash-8090-Yadav) [inline .github/ISSUE_TEMPLATE/improvements.md:13]: write Domain only in place of Domain of Docs in line 13

File: .github/ISSUE_TEMPLATE/add-project.md [hunk 8]
```diff
@@ -16,7 +16,7 @@ assignees: ''
 
 ### Doc Domain
```
Comment (Vikash-8090-Yadav) [inline .github/ISSUE_TEMPLATE/add-project.md:17]: write Domain only in place of Doc Domain line no 17

**Comment by Vikash-8090-Yadav** [comment]
**Update all these chnages ... ALL SET TO MERGE** 
:smile:

**Comment by surjyaneeh** [comment]
> **Update all these chnages ... ALL SET TO MERGE** 😄

Done

**Comment by Vikash-8090-Yadav** [comment]
Cool :smile:

**Pull request merged**